src/Argument.h

```diff
@@ -56,13 +56,11 @@ struct Argument {
      * By default, they are left unset, implying "no default, no min, no max". */
     Expr def, min, max;
 
-  int stride_multiples[4];
```

Comment:
I think that is exactly what is happening. See my comment on the PR (not just the file). I think maybe this is a situation where you need to use git rebase to fix the history.

Again, sorry to be a pain about this.